OstelcoStyles/OstelcoLabel.swift

```diff
@@ -47,6 +47,26 @@ open class OstelcoLabel: UILabel {
         super.prepareForInterfaceBuilder()
         self.commonInit()
     }
+    
+    public func setFullText(_ fullText: String, withBoldedPortion boldedPortion: String) {
```

Comment:
Something to be aware of when pulling this into the superclass: If  the label's `appFont` is already bold, this won't actually have any effect. 

It's fine to leave it for now though - I'll likely change this up to allow passing in attributes so we can use this for the linky text, and if it takes arbitrary attributes, then you can easily pass in some other differentiating attributes if the font is bold.

Reply:
I will admit that this is me being lazy, in zeplin, the text is actually marked as body text, which is the class that contained the `setFullText`. So I will move the `setFullText` back and change the class I use for both top and bottom text label.

For future references, would it be an idea to create a subclass of `OstelcoLabel` where we could move the `setFullText` eg `AttributedOstelcoLabel`, then for each label that requires anything with attributed string, subclass from the `AttributedOstelcoLabel` instead of `OstelcoLabel`?

Reply:
The changes in #231 make this more general and allow it to be used with any label, so if you want to just leave this I'll deal with it in merge funtimes